### PR TITLE
Blog post about Prefix' "Chatting with the conda community"

### PR DESCRIPTION
**Pull request by wolfv**
### Description

- [x] If I have added a new page to `learn/` or `community/`, I have added it to the corresponding `_sidebar.json` file.

### CODE REVIEW

**Review by jezdez — CHANGES_REQUESTED** [review]
Thanks for openning the PR, the video is presented as a conda community update but it's discussing conda-forge and pixi, neither of which are conda community projects.

This wasn't discussed with the conda communications team before as a new inititive. So I have to say I'm a little flummoxed about it, it's not as if I haven't been trying to faciliate the participation of @wolfv and his company, why presenting this as a done deal?

@wolfv Was this the topic you brought up recently when you asked to "record a podcast" with me?

A few things that I think need to happen before we merge and publish it:

- the prefix.dev GmbH heavy theming throws me off, just like having it hosted on a company youtube channel. This is fixable by setting up a youtube channel and upload the video there.
- pixi is as you are well aware _not_ a conda community project, but a commercial project by @wolfv's company, which makes this framing as a conda community update a little unsavory in my book. This is after all the type of vendor focus that the conda governance policy was set up to prevent, and that I've personally tried very hard for us to walk away from.
- Neither @wolfv nor @ocefpaf are members of the conda communications team and it hadn't been brought up with the conda steering council before recording it, so I'd be interested to hear more about the plans for this community update (E.g. how often? What is the focus? How does this relate to the conda community sync meeting? Who will be the target audience?)

**Comment by wolfv** [comment]
Yes, I had this format in mind when I talked to you about a Podcast (but more with the NumFOCUS transition in mind and the revamping of the whole conda-community effort).

I think both conda-forge and pixi are very interesting topics to the conda community. Not sure why we wouldn't want to hear about them :) Just as we would want to hear from Docker, Nix or Homebrew.

I think most people will remember more "conda" than "prefix". We are not selling "pixi" so it's not really fair to classify it as a "commercial product".

We are hoping to do this kind of recordings roughly once or twice a month. Probably more once a month. Topics should be anything touching the conda community:

- Talking about mamba with @SylvainCorlay
- Talking about conda recipe manager e.g. with Schuyler
- Talking about rez & conda with e.g. with JC
- conda-forge infrastructure and bots e.g. with Matt Becker

... and more ideas along those lines (no one knows about these ideas yet, so just putting them out here for now).

If you want we can come up with a different name of course. It would sadden me (because I am hoping to reinvigorate the community) but it's no problem!

**Comment by wolfv** [comment]
Some ideas for alternative names:

- "Conda Convo"
- "All things Conda"
- "Package Party"

Would one of those alleviate your concerns @jezdez?

**Comment by dashagurova** [comment]
@wolfv I love the idea of podcast/chats for the conda ecosystem! If the goal is to reinvigorate the conda community, I think that starting a dedicated conda ecosystem YouTube channel would make more sense. We could host the videos there and then share them through the official conda website and social media channels. And I would be happy to volunteer to help and collaborate on this! I’ve had a similar idea of starting conda community podcast/chats someday, where we talk about/with the projects from the conda ecosystem, including pixi, mamba, conda, conda-forge (many more from conda and conda incubator) and related or interesting topics (like packaging in general).

**Comment by wolfv** [comment]
@dashagurova - great! If you make the channel I can move the video there. Maybe you can give me content creator rights on that channel?

I wouldn't mind having other hosts on this "show", either :)

**Comment by jezdez** [comment]
Hey @wolfv, sorry for the slow reply, I like the idea of doing more outreach for the amazing things that happen in our ecosystem, despite my worries above, my main focus remains the sustainability of the community and starting a whole new project without plan sounds like we'd be setting us up for failure. Thanks for chiming in @dashagurova as well!

Could we start putting together a plan in the conda/communications repo ([as an issue](https://github.com/conda/communications/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen)?) with a publication schedule, roles and purposes, so that we can meaningfully say this comes from the community? (that's why we set up the communications team originally after all) I'm a little uncertain how else to make sure this doesn't end up being a project that fizzles out after a short while. I do appreciate your willingness do the work @wolfv, maybe I could encourage you to participate?

Regarding naming, it really depends on the format, is this mostly going to be 1:1 converstaions like fireside chats? Would we consider a panel or hosted talks?

File: blog/2024-08-21-chatting-with-the-conda-community.mdx
Comment (dashagurova) [inline blog/2024-08-21-chatting-with-the-conda-community.mdx:5]:
```suggestion
tags: [announcement]
```

File: blog/news.json
Comment (dashagurova) [inline blog/news.json:10]:
```suggestion
      "announcement"
```

**Review by jezdez — COMMENTED** [review]
Thanks for updating @wolfv!

**Comment by wolfv** [comment]
I updated the date. IMO we can merge this one if people are fine with it

File: blog/2024-09-02-chatting-with-the-conda-community.mdx
Comment (wolfv) [reply blog/2024-09-02-chatting-with-the-conda-community.mdx]: Done!

File: static/img/blog/2024-08-21-chatting-with-conda-community/banner.png
Comment (wolfv) [reply static/img/blog/2024-08-21-chatting-with-conda-community/banner.png:1]: I moved it to a differnet folder.

**Comment by wolfv** [comment]
I'll merge this tomorrow if no one else gets to it. :)

**Review by jezdez — APPROVED** [review]

File: blog/2024-09-12-chatting-with-the-conda-community.mdx
Comment (jaimergp) [inline blog/2024-09-12-chatting-with-the-conda-community.mdx:3]: Sorry one last thing, this should include the date:

```suggestion
slug: "2024-09-12-chatting-with-the-conda-community-1"
```

File: blog/news.json
Comment (jaimergp) [inline blog/news.json:4]:
```suggestion
    "slug": "2024-09-12-chatting-with-the-conda-community-1",
```

**Comment by jaimergp** [comment]
Thanks!

**Pull request merged**
